data/saas/config/attentive_config.yml

```diff
@@ -23,15 +23,15 @@ saas_config:
     path: /v1/me
 
   endpoints:
-    - name: delete_request
+    - name: user
       requests:
         delete:
           method: POST
           path: /v1/privacy/delete-request
           body: |
             {
-             "subjectEmail": "<email>",
-             "subjectPhone": "<phone_number>",
+             "subjectEmail": "<email?>",
+             "subjectPhone": "<phone_number?>",
```

Comment:
We have this `?` notation in our placeholders to indicate that this value might be `null` but it's ok! I added this here so there aren't any errors if either `email` or `phone_number` are missing